src/dataSources/cloudFirestore/session.js

```diff
@@ -186,15 +187,25 @@ const session = dbInstance => {
     communitySlug,
     statuses,
     orderBy,
-    pagesize,
+    filter,
+    asOfDate,
+    pageSize,
     cursor,
   }) {
     dlog('findByCommunityWithStatuses %s, %o', communitySlug, statuses);
     const slimslug = communitySlug.trim().toLowerCase();
     const inStatus = validateStatuses(statuses);
+    const truePSize = Math.min(pageSize || 20, 100); // max page: 100
+    let allOrderBy = 'desc';
+    if (orderBy === 'START_TIME_ASC') allOrderBy = 'asc';
+
     let startTimeOrder = 'asc';
-    if (orderBy === 'START_TIME_DESC') startTimeOrder = 'desc';
-    const truePSize = Math.min(pagesize || 20, 100); // max page: 100
+    if (filter === 'PAST') {
```

Comment:
this is coming from the graphql enum right? Because of that it should be upper. otherwise we might want to toUpperCase() it just to be sure.